### PR TITLE
Match report role

### PR DESCRIPTION
**Pull request by csconner1998**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -37,6 +37,13 @@
 leaderboardChannel = ""
 msgID = ""
 
+async def isRole(user,role,message):
+    role = discord.utils.find(lambda r: r.name == role, message.guild.roles)
+    if role in user.roles:
+        return True
+    else:
+        return False
```
Comment (GNMoseke) [inline bot.py]: nit
```python
return role in user.roles
```

File: bot.py [hunk 2]
```diff
@@ -85,8 +95,11 @@ async def checkWinStr(reaction,user):
     await leaderboard.updateLeaderboard(reaction.message.channel, leaderboardMsgs=leaderboardMsgs, leaderboardChannel=leaderboardChannel, db_handler=db_handler)
     
 async def checkStart(message):
-    if len(message.reactions) < 12:
-        return ""
+    count = 0
+    for i in message.reactions:
+        count += i.count
```
Comment (GNMoseke) [inline bot.py]: oh nice, this is way better than the list comp I was trying to do

Comment (GNMoseke) [reply bot.py]: but my list comprehension loving self has come up with a one liner:
```python
count = sum([reaction.count for reaction in message.reactions])
```

File: bot.py (hunk 1 above)
Comment (csconner1998) [reply bot.py]: Fixed

**Review by GNMoseke — APPROVED** [review]

**Pull request merged**